cloudbuild.yaml

```diff
@@ -0,0 +1,16 @@
+steps:
+  - id: version
+    name: 'gcr.io/cloud-builders/go'
+    args: ['version']
+    env: ['GOPATH=/gopath', 'GO111MODULE=on']
+  - id: test
+    name: 'gcr.io/cloud-builders/go'
+    args: ['test', '-v', '-coverprofile=coverage.out', '-covermode=atomic' , './...']
+    env: ['GOPATH=/gopath', 'GO111MODULE=on']
+  - id: cover
+    name: 'gcr.io/cloud-builders/go'
+    env: ['GOPATH=/gopath', 'GO111MODULE=on']
+    entrypoint: 'sh'
+    args:
+      - -c 
+      - go tool cover -func=coverage.out | grep total | awk '/^total:/ { gsub(/%$/, ""); if ($3 < 100.0) {print "FAIL - Coverage below 100%"; exit(1)} else {print "Coverage 100%" } }'
```

Comment:
The `| grep total` looks to be redundant as awk is already matching that.

Reply:
`grep` is not redundant in case `coverage.out` is not created; in that case `awk` alone has exit code 0 with `file not found` message. grep forces the build step to fail. i was hoping you'd show me a better solution :wink: